src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -263,7 +268,7 @@ const TimeSeriesCard = ({
     const defaultFillColor = !isEditable ? originalFillColor : '#f3f3f3';
     if (!isNil(data)) {
       const matchingAlertRange = findMatchingAlertRange(alertRanges, data);
-      return matchingAlertRange ? matchingAlertRange.color : defaultFillColor;
+      return matchingAlertRange?.length > 0 ? matchingAlertRange[0].color : defaultFillColor;
```

Comment:
Is our babel config set to enable optional chaining?

Reply:
it is!